vms/avm/vm_test.go

```diff
@@ -1121,63 +1116,6 @@ func TestTxVerifyAfterGet(t *testing.T) {
 	require.ErrorIs(err, database.ErrNotFound)
 }
 
-func TestTxVerifyAfterVerifyAncestorTx(t *testing.T) {
```

Comment:
This test was explicitly checking for the functionality we are removing:
- Verification of a tx that depends on the state of a processing tx.